### DISCUSSION TITLE
[milvus2.0 cluster]  when collection loadding or release Causing cluster nodes to hang up

### DISCUSSION THREAD

**Discussion by zbhuang666** [milvus-io/milvus#16561] [Q&A and General discussion]
[FATAL] [logutil.go:102] [panic] [recover="\"Failed to create consumer by-dev-rootcoord-dml_99, error = All attempts results:\\nattempt #1:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #2:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #3:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #4:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #5:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #6:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #7:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #8:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #9:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #10:server error: ConsumerBusy: Exclusive consumer is already connected\\nattempt #11:server error: ConsumerBusy: Exclusive consumer is already connected\\n\""] [stack="[github.com/milvus-io/milvus/internal/util/logutil.LogPanic\n\t/go/src/github.com/milvus-io/milvus/internal/util/logutil/logutil.go:102\nruntime.gopanic\n\t/usr/local/go/src/runtime/panic.go:965\ngithub.com/milvus-io/milvus/internal/msgstream.(*mqMsgStream).AsConsumerWithPosition\n\t/go/src/github.com/milvus-io/milvus/internal/msgstream/mq_msgstream.go:165\ngithub.com/milvus-io/milvus/internal/msgstream.(*mqMsgStream).AsConsumer\n\t/go/src/github.com/milvus-io/milvus/internal/msgstream/mq_msgstream.go:131\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).unsubscribe\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:369\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).cleanUpAndDelete\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:607\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).processAck\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:578\ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).watchChannelStatesLoop\n\t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:661](http://github.com/milvus-io/milvus/internal/util/logutil.LogPanic/n/t/go/src/github.com/milvus-io/milvus/internal/util/logutil/logutil.go:102/nruntime.gopanic/n/t/usr/local/go/src/runtime/panic.go:965/ngithub.com/milvus-io/milvus/internal/msgstream.(*mqMsgStream).AsConsumerWithPosition/n/t/go/src/github.com/milvus-io/milvus/internal/msgstream/mq_msgstream.go:165/ngithub.com/milvus-io/milvus/internal/msgstream.(*mqMsgStream).AsConsumer/n/t/go/src/github.com/milvus-io/milvus/internal/msgstream/mq_msgstream.go:131/ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).unsubscribe/n/t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:369/ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).cleanUpAndDelete/n/t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:607/ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).processAck/n/t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:578/ngithub.com/milvus-io/milvus/internal/datacoord.(*ChannelManager).watchChannelStatesLoop/n/t/go/src/github.com/milvus-io/milvus/internal/datacoord/channel_manager.go:661)"]

[ERROR] [client.go:254] ["datacoord ClientBase ReCall grpc second call get error "] [error="err: context deadline exceeded\n, /go/src/[github.com/milvus-io/milvus/internal/util/trace/stack_trace.go:51](http://github.com/milvus-io/milvus/internal/util/trace/stack_trace.go:51) [github.com/milvus-io/milvus/internal/util/trace.StackTrace\n/go/src/github.com/milvus-io/milvus/internal/util/grpcclient/client.go:253](http://github.com/milvus-io/milvus/internal/util/trace.StackTrace/n/go/src/github.com/milvus-io/milvus/internal/util/grpcclient/client.go:253) [github.com/milvus-io/milvus/internal/util/grpcclient.(*ClientBase).ReCall\n/go/src/github.com/milvus-io/milvus/internal/distributed/datacoord/client/client.go:111](http://github.com/milvus-io/milvus/internal/util/grpcclient.(*ClientBase).ReCall/n/go/src/github.com/milvus-io/milvus/internal/distributed/datacoord/client/client.go:111) [github.com/milvus-io/milvus/internal/distributed/datacoord/client.(*Client).GetComponentStates\n/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:59](http://github.com/milvus-io/milvus/internal/distributed/datacoord/client.(*Client).GetComponentStates/n/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:59) [github.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentStates.func1\n/go/src/github.com/milvus-io/milvus/internal/util/retry/retry.go:34](http://github.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentStates.func1/n/go/src/github.com/milvus-io/milvus/internal/util/retry/retry.go:34) [github.com/milvus-io/milvus/internal/util/retry.Do\n/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:83](http://github.com/milvus-io/milvus/internal/util/retry.Do/n/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:83) [github.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentStates\n/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:98](http://github.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentStates/n/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:98) [github.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentHealthy\n/go/src/github.com/milvus-io/milvus/internal/distributed/proxy/service.go:276](http://github.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentHealthy/n/go/src/github.com/milvus-io/milvus/internal/distributed/proxy/service.go:276) [github.com/milvus-io/milvus/internal/distributed/proxy.(*Server).init\n/go/src/github.com/milvus-io/milvus/internal/distributed/proxy/service.go:171](http://github.com/milvus-io/milvus/internal/distributed/proxy.(*Server).init/n/go/src/github.com/milvus-io/milvus/internal/distributed/proxy/service.go:171) [github.com/milvus-io/milvus/internal/distributed/proxy.(*Server).Run\n/go/src/github.com/milvus-io/milvus/cmd/components/proxy.go:49](http://github.com/milvus-io/milvus/internal/distributed/proxy.(*Server).Run/n/go/src/github.com/milvus-io/milvus/cmd/components/proxy.go:49) [github.com/milvus-io/milvus/cmd/components.(*Proxy).Run\n](http://github.com/milvus-io/milvus/cmd/components.(*Proxy).Run/n)"] [stack="[github.com/milvus-io/milvus/internal/util/grpcclient.(*ClientBase).ReCall\n\t/go/src/github.com/milvus-io/milvus/internal/util/grpcclient/client.go:254\ngithub.com/milvus-io/milvus/internal/distributed/datacoord/client.(*Client).GetComponentStates\n\t/go/src/github.com/milvus-io/milvus/internal/distributed/datacoord/client/client.go:111\ngithub.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentStates.func1\n\t/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:59\ngithub.com/milvus-io/milvus/internal/util/retry.Do\n\t/go/src/github.com/milvus-io/milvus/internal/util/retry/retry.go:34\ngithub.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentStates\n\t/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:83\ngithub.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentHealthy\n\t/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:98\ngithub.com/milvus-io/milvus/internal/distributed/proxy.(*Server).init\n\t/go/src/github.com/milvus-io/milvus/internal/distributed/proxy/service.go:276\ngithub.com/milvus-io/milvus/internal/distributed/proxy.(*Server).Run\n\t/go/src/github.com/milvus-io/milvus/internal/distributed/proxy/service.go:171\ngithub.com/milvus-io/milvus/cmd/components.(*Proxy).Run\n\t/go/src/github.com/milvus-io/milvus/cmd/components/proxy.go:49\ngithub.com/milvus-io/milvus/cmd/roles.(*MilvusRoles).runProxy.func1\n\t/go/src/github.com/milvus-io/milvus/cmd/roles/roles.go:147](http://github.com/milvus-io/milvus/internal/util/grpcclient.(*ClientBase).ReCall/n/t/go/src/github.com/milvus-io/milvus/internal/util/grpcclient/client.go:254/ngithub.com/milvus-io/milvus/internal/distributed/datacoord/client.(*Client).GetComponentStates/n/t/go/src/github.com/milvus-io/milvus/internal/distributed/datacoord/client/client.go:111/ngithub.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentStates.func1/n/t/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:59/ngithub.com/milvus-io/milvus/internal/util/retry.Do/n/t/go/src/github.com/milvus-io/milvus/internal/util/retry/retry.go:34/ngithub.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentStates/n/t/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:83/ngithub.com/milvus-io/milvus/internal/util/funcutil.WaitForComponentHealthy/n/t/go/src/github.com/milvus-io/milvus/internal/util/funcutil/func.go:98/ngithub.com/milvus-io/milvus/internal/distributed/proxy.(*Server).init/n/t/go/src/github.com/milvus-io/milvus/internal/distributed/proxy/service.go:276/ngithub.com/milvus-io/milvus/internal/distributed/proxy.(*Server).Run/n/t/go/src/github.com/milvus-io/milvus/internal/distributed/proxy/service.go:171/ngithub.com/milvus-io/milvus/cmd/components.(*Proxy).Run/n/t/go/src/github.com/milvus-io/milvus/cmd/components/proxy.go:49/ngithub.com/milvus-io/milvus/cmd/roles.(*MilvusRoles).runProxy.func1/n/t/go/src/github.com/milvus-io/milvus/cmd/roles/roles.go:147)"]

**Comment by yhmo** [comment]
I have created an issue: https://github.com/milvus-io/milvus/issues/16566, it will be assigned to developers to analyse.

**Comment by yhmo** [comment]
@zbhuang666 Could you tell me the version of your milvus?
